obspy/clients/fdsn/routers/fedcatalog_client.py

```diff
@@ -56,15 +57,18 @@
     inventory_to_bulkrequests, stream_to_bulkrequests)
 
 
+# ignore InsecureRequestWarning
+# https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
```

Comment:
hmm.. so if we don't do certificate verification, does it make sense to bother requesting from https://?  @QuLogic maybe you can comment on this..? I don't have a lot of experience with this.